frontend/src/components/TransactionItem.tsx

```diff
@@ -88,7 +88,11 @@ function TransactionItem({ tx }: Props) {
 
   const eventId = tx.metadata?.nostr?.tags?.find((t) => t[0] === "e")?.[1];
 
-  const description = tx.description || tx.metadata?.comment;
+  const bolt12Offer = tx.metadata?.offer;
+  const isBolt12Offer = !!bolt12Offer?.id;
+
+  const description =
```

Comment:
If the offer has a description AND the payer pays with a payer note, will the payer note appear somewhere?

Can we show the payer note in the modal when you click on the transaction?

Reply:
hmm, I see it is, but the description is not set for received payments